AdminWebsite/AdminWebsite/ClientApp/src/app/booking/add-staff-member/add-staff-member.component.ts

```diff
@@ -0,0 +1,115 @@
+import { Component, EventEmitter, Input, OnDestroy, OnInit, Output } from '@angular/core';
+import { ParticipantModel } from 'src/app/common/model/participant.model';
+import { AddParticipantBaseDirective } from 'src/app/booking/add-participant-base/add-participant-base.component';
+import { Router } from '@angular/router';
+import { BookingService } from 'src/app/services/booking.service';
+import { Logger } from 'src/app/services/logger';
+import { VideoHearingsService } from 'src/app/services/video-hearings.service';
+import { Subscription } from 'rxjs';
+import { Constants } from 'src/app/common/constants';
+
+@Component({
+    selector: 'app-add-staff-member',
+    templateUrl: './add-staff-member.component.html',
+    styleUrls: ['./add-staff-member.component.css']
+})
+export class AddStaffMemberComponent extends AddParticipantBaseDirective implements OnInit, OnDestroy {
+    @Input() isShowErrorSummary: boolean;
+    @Output() staffMember = new EventEmitter<ParticipantModel>();
+    @Output() isStaffMemberValid = new EventEmitter<boolean>();
+
+    $subscriptions: Subscription[] = [];
+
+    isSubscribedToEmailChanges = false;
+
+    constructor(
+        protected bookingService: BookingService,
+        protected router: Router,
+        protected videoHearingService: VideoHearingsService,
+        protected logger: Logger
+    ) {
+        super(bookingService, router, videoHearingService, logger);
+    }
+
+    ngOnInit(): void {
+        this.checkForExistingRequest();
+        this.initialiseForm();
+
+        this.setupStaffMemberValidityEmissionOnFormValueChange();
+        this.setupStaffMemberEmissionWhenValid();
+    }
+
+    ngOnDestroy(): void {
+        this.$subscriptions.forEach(subscription => {
+            if (subscription) {
+                subscription.unsubscribe();
+            }
+        });
+    }
+
+    private checkForExistingRequest() {
+        this.hearing = this.videoHearingService.getCurrentRequest();
+    }
+
+    initialiseForm() {
+        super.initialiseForm();
+        this.form.removeControl('interpreterFor');
```

Comment:
This is a little odd but should be fine for now. Is this part of the tech debt?

Reply:
Yep, was part of the last conversation.